cluster/crds/pkg.crossplane.io_deploymentruntimeconfigs.yaml

```diff
@@ -4211,8 +4211,6 @@ spec:
                                     ip:
                                       description: IP address of the host file entry.
                                       type: string
-                                  required:
```

Comment:
i wouldn't expect to see this field in the `DeploymentRuntimeConfig` CRD get updated to no longer be required, given the change in this PR is to bump `github.com/google/go-containerregistry`.

I believe this is the field that is "no longer required" in the CRD: 
https://docs.crossplane.io/v1.16/api/#DeploymentRuntimeConfig-spec-deploymentTemplate-spec-template-spec-hostAliases-ip

This looks suspicious so far 🔍